melview.py

```diff
@@ -411,6 +427,16 @@ def set_mode(self, mode):
             return False
         return self._send_command('MD{}'.format(MODE[mode]))
 
+    def enable_zone(self, zoneid):
+        """ Turn on a zone.
+        """
+        return self._send_command('Z' + str(zoneid) + '1')
```

Comment:
Can you do these with format strings - I'm wanting to change everything over eventually.

```python
f"z{zoneid}1"
```